src/analysis/special_functions/trigonometric/angle.lean

```diff
@@ -123,6 +123,19 @@ by rw [eq_comm, eq_neg_self_iff]
 lemma neg_ne_self_iff {θ : angle} : -θ ≠ θ ↔ θ ≠ 0 ∧ θ ≠ π :=
 by rw [←not_or_distrib, ←neg_eq_self_iff.not]
 
+lemma two_nsmul_eq_pi_iff {θ : angle} : (2 : ℕ) • θ = π ↔ (θ = (π / 2 : ℝ) ∨ θ = (-π / 2 : ℝ)) :=
```

Comment:
Some files prefer `-(π / 2)`, some prefer `-π / 2 = (-π) / 2`. Do we have a simp normal form?

Reply:
I'm not aware of a simp normal form (`neg_div` isn't a simp lemma in either direction), so I used `-π / 2` as it's fewer characters (i.e. treated the precedence of `/` and unary `-` as establishing a preferred form in the absence of a strong reason to choose a particular form).